src/components/ProfileInformation/ProfileInformation.tsx

```diff
@@ -6,29 +6,37 @@ import Copy from '../../assets/icons/Copy.svg'
 import People from '../../assets/icons/People.svg'
 import Share from '../../assets/icons/Share.svg'
 import Wallet from '../../assets/icons/Wallet.svg'
+import { Props } from './ProfileInformation.types'
 import styles from './ProfileInformation.module.css'
 
-const ProfileInformation = () => {
-  return (
+const ProfileInformation = (props: Props) => {
+  const { profile, isLoggedInProfile = false } = props
+
+  const profileData = profile?.avatars[0]
+
+  return profileData ? (
```

Comment:
If I'm not wrong, there are profiles without avatars. Those profiles are the ones that didn't interact with the explorer. For those addresses we need to show the `ProfileInformation` with the Unnamed name and others.